src/Prefabs/Vegetation/Tree/t_pinus_sylvestris_2s.et

```diff
@@ -0,0 +1,16 @@
+Tree : "{388AE316D09D0680}Prefabs/Vegetation/Core/Tree_Base.et" {
```

Comment:
Why is this existing tree prefab specifically overridden?

Reply:
This was the tree I was testing logging on. But since its on the Tree_Base now this is indeed no longer needed!